README.md

```diff
@@ -173,6 +173,7 @@ $ cat pixi.toml | tail -2
 [dependencies]
 numpy = ">=2.0.1,<3"
 ```
+`pixi` can install any package, so long as the package exists in one of the channels listed in the `pixi.toml` file. This means that `pixi` is not just able to install standard python packages such as `numpy`, `scipy`, and `matplotlib`; but also packages such as `cmake` for use in compiling c++ code!
```

Comment:
Thank you for this added information, could you please change the semi-colon to a comma and drop the '!' as it does not match the style of the rest of the project.

"...such as `numpy`, `scipy`, and `matplotlib`, but also packages such as `cmake` for use in compiling c++ code."